.eslintrc.json

```diff
@@ -3,8 +3,9 @@
       "browser": true,
       "es6": true
     }, 
-    "globals":{
-      "Product": false
+    "globals": {
+      "Product": false,
```

Comment:
Added our `Product` and `Chart` to our eslint globals list - this will stop it from erroring out when separated into another file